apiserver/adapters/in_memory_adapter.go

```diff
@@ -0,0 +1,59 @@
+package adapters
+
+import (
+	"errors"
+	"github.com/astanishevskyi/http-server/apiserver/models"
+	"math/rand"
+)
+
+type UserStorage struct {
+	UsersSlice []models.User `json:" users"`
```

Comment:
Concurrent writes to this slice will be unsafe without using synchronization technics. Think why.

Reply:
Fixed